#!/usr/bin/env python
PACKAGE = "collab_vehicle_behavior"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("hue_low",                  double_t,   0, "Hue Low Value",         0.0,  0.0,    360.0)
gen.add("hue_high",                 double_t,   0, "Hue High Value",        32.0,  0.0,    360.0)
gen.add("sat_low",                  double_t,   0, "Sat Low Value",         150.0,   0.0,    255.0)
gen.add("sat_high",                 double_t,   0, "Sat High Value",        255.0,  0.0,    255.0)
gen.add("lum_low",                  double_t,   0, "Lum Low Value",         127.5,    0.0,    255.0)
gen.add("lum_high",                 double_t,   0, "Lum High Value",        255.0,  0.0,    255.0)
gen.add("dilation_size",            double_t,   0, "Dialation size",        0,    0.0,    3.0)
gen.add("largest_area",             double_t,   0, "Largest Area",          1000.0, 0.0,    50000.0)


exit(gen.generate(PACKAGE, "collab_vehicle_behavior", "Commander"))